handler/rest.go

```diff
@@ -132,4 +132,43 @@ func (h ArticleHandler) Delete(c echo.Context) (err error) {
 	}
 
 	return c.JSON(http.StatusCreated, deletedID)
+}
+
+func (h ArticleHandler) Update(c echo.Context) (err error){
+	articleID := c.Param("id")
+	var item models.Article
+	err = c.Bind(&item)
+	if err != nil {
+		resp := ErroResponse{
+			Message: err.Error(),
+		}
+		return c.JSON(http.StatusUnprocessableEntity, resp)
+	}
+
```

Comment:
After binding the value, re-assign the ID.

`item.ID = articleID`

Reply:
Done